style.css

```diff
@@ -0,0 +1,3 @@
+h1{
+    color: rgb(155, 39, 39);
+}
```

Comment:
- [ ] Kindly adhere to the project requirement as much as possible, I can see as per requirement you must need to add style on class for your `header` **h1**. So please add the class name for your *h1 tag* in your `index.html` file and add that class name at the place of `h1`.